server/migrations/20200805094217-create-historicalValidatorStats-table.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('HistoricalValidatorStats', {
+      id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true }, //primary-key
+      stash_id: { type: DataTypes.STRING, allowNull: false, references: { model: 'Validators', key: 'stash' } },
+      block: { type: DataTypes.STRING, allowNull: false }, // blocknumber
+      exposure: { type: DataTypes.JSON, allowNull: false },
+      commissionPer: { type: DataTypes.FLOAT, allowNull: false },
+      preferences: { type: DataTypes.INTEGER, allowNull: false },// preferences - ValidatorPrefs
```

Comment:
we should remove this since the preferences are only commission: https://github.com/paritytech/substrate/blob/master/frame/staking/src/lib.rs#L447.